cypress/e2e/cloud/dashboardsView.test.ts

```diff
@@ -1,4 +1,4 @@
-describe('Dashboard', () => {
+describe.skip('Dashboard', () => {
   beforeEach(() =>
```

Comment:
Since we're skipping more tests we need these on the list - i'll add 'em.

Reply:
NVM, Denise added  them to the tracking issue already. 🥳 
https://github.com/influxdata/ui/issues/6562